src/lib/features/playback-controls/ui/scrubbing-bar.svelte

```diff
@@ -23,10 +23,12 @@
     type="range"
     min={0}
     max={duration}
-    step={0.01}
+    step={0.001}
     value={position}
     aria-valuetext={formatDuration(position)}
-    on:input={(e) => dispatch('scrub', { position: parseInt(e.currentTarget.value) })}
+    on:pointerup={() => dispatch('scrub', { dragging: false, position })}
```

Comment:
thought (non-blocking): looks good in that it probably works, but I can't shake the feeling of wanting something more elegant. My points of concern with this solution are the keyboard accessibility and the support of browsers regarding pointer events. The latter is kind of a nitpick since even IE supports pointer events, but the former one is a valid concern.

The crap thing is that I have no clue myself about how to implement this in a better way. Spent a good 10 minutes thinking of a way, and nothing came of it. For future reference, here's [a minimal reproduction of this issue](https://svelte.dev/repl/10118a4906d1479ab1ae9aae218ceeb2?version=3.46.4) in the Svelte REPL.